.devcontainer/k3d.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: k3d.io/v1alpha4
+kind: Simple
+servers: 1
+network: k3d
+kubeAPI:
+  hostIP: "0.0.0.0"
+  hostPort: "6443"
+volumes:
```

Comment:
Don't think we need prometheus and grafana volumes

Reply:
Good point. I can remove these.